source/includes/cli-reference/_projectfile.html.md

```diff
@@ -53,6 +53,13 @@ There must be at least one valid path in the `files` list.
 points to a pre-configured simulation container inside the platform. The
 `command` field describes the command to run to start the simulator.
 
+Current list of supported simulators for Docker cloud-hosted training:
+
+ * [`openai.gym`][1]
+ * [`bonsai.python`][2]
+ * [`bonsai.energyplus`][3]

```

Comment:
I'm not sure how much it makes sense to link `bonsai-gym-common`, `bonsai-python` and `energyplus-sample` repositories directly to the list of supported simulators here. It might be better to link the images themselves on quay.io?

Specifically, the Dockerfiles we use to build these images isn't released anywhere (which is what one would normally expect to see from a github link in the context of a docker container).